tools/themis/src/rules.rs

```diff
@@ -0,0 +1,329 @@
+use std::collections::HashMap;
+
+use super::{style, utils, Error, Expected, PackageOutcome, Workspace};
+
+/// ensure all crates have the `publish = <true/false>` specification
```

Comment:
I with in general, we prefer to start comment for functions with an Upper Case letter.